docs/source/complex_numbers.rst

```diff
@@ -129,9 +129,33 @@ Autograd
 
 PyTorch supports autograd for complex tensors. The gradient computed is the Conjugate Wirtinger derivative,
 the negative of which is precisely the direction of steepest descent used in Gradient Descent algorithm. Thus,
-all the existing optimizers work out of the box with complex parameters. For more details,
+all the existing optimizers can be implemented to work out of the box with complex parameters. For more details,
 check out the note :ref:`complex_autograd-doc`.
 
+
+Optimizers
+----------
+
+Semantically, stepping through a PyTorch optimizer with complex parameters is equivalent to stepping through the
+same optimizer on the :func:`torch.view_as_real` equivalent of the complex params. More concretely:
+
+::
+
+     >>> params = [torch.rand(2, 3, dtype=torch.complex64) for _ in range(5)]
+     >>> real_params = [torch.view_as_real(p) for p in params]
+
+     >>> complex_optim = torch.optim.AdamW(params)
+     >>> real_optim = torch.optim.AdamW(real_params)
+
+
+`real_optim` and `complex_optim` will compute the same updates on the parameters. Note that the :func:`torch.view_as_real`
+equivalent will convert a complex tensor to a real tensor with shape :math:`(..., 2)`, which is distinct from splitting
```

Comment:
I think we are over-indexing this explanation on that very small details.
I would split this to a second paragraph that covers:
- There can be numerical discrepancies between the two (complex vs view as real-ed) just like capturable/foreach/etc Linking to https://pytorch.org/docs/stable/notes/numerical_accuracy.html for details
- The note about one Tensor vs two Tensors. I think it's worth mentioning that this will only make a different for optimizers that do per-Tensor reductions (which we don't have any of) and we don't define this behavior yet, open an issue if you have a use case that needs this behavior to be defined.